prism/static/js/src/studio.js

```diff
@@ -0,0 +1,33 @@
+/* Javascript for PrismXBlock. */
+function PrismXBlockEditBlock(runtime, element) {
+    var $element = $(element);
+
+    var lngSelect = document.getElementById('language');
+    var thmSelect = document.getElementById('theme');
+
+    var codeMirror = CodeMirror.fromTextArea(document.querySelectorAll('textarea#code-textarea')[0], {
+        lineNumbers: true,
+        indentWithTabs: true,
+        matchBrackets: true,
+        lineWrapping: true,
+    });
+
+    $(element).find('.save-button').bind('click', function(){
+        var handlerUrl = runtime.handlerUrl(element, 'studio_submit'); 
+        var data = {
+            display_name: $(element).find('input[name=display_name]').val(),
+            code_data: $(element).find('#code-textarea').val(),
```

Comment:
**fix:** I've used this StackOverflow answer: https://stackoverflow.com/a/20267975

```suggestion
            code_data: codeMirror.getValue(),
```